### ISSUE TITLE
[FR] Adafruit featheresp32

### ISSUE THREAD

**Issue by mjmeans** [MarlinFirmware/Marlin#22974]
### Are you looking for hardware support?

'feather32esp" is a supported env in PlatformIO.

### Describe the feature you want

'feather32esp" is a supported env in PlatformIO. Is it possible to build Marlin for this board? What motherboard could work with it? I am only needed X/Y, but not Z since my project would be for like a pencil drawing platform and powered by an 18650 battery pack.

**Comment by thisiskeithb** [comment]
Some ESP32 boards are supported, but the  Adafruit HUZZAH32 / ESP32 Feather isn’t one of them:

https://github.com/MarlinFirmware/Marlin/blob/3c94f2f1c33395e45f3b2456f97eb64310b41000/Marlin/src/core/boards.h#L417-L427

**Comment by mjmeans** [comment]
Yes, I've seen that. What would I need to do to add the Adafruit board? I'm not very familiar with Arduino coding and even less familiar with how Marlin works under the hood.

**Comment by mjmeans** [comment]
I've tried adding this to esp.ini:

```
[env:featheresp32]
platform               = espressif32@2.1.0
extends                = env:esp32
board                  = featheresp32

```
And setting BOARD to `BOARD_ESPRESSIF_ESP32`. It compiles but will not upload to the HUZZAH32.
